src/blog/2024/08/using-mqtt-sparkplugb-with-node-red.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Using MQTT Sparkplug B with Node-RED
+subtitle: A Practical Guide to integrating MQTT Sparkplug B with Node-RED
+description: Explore how MQTT Sparkplug B enhances MQTT with standardized data formats, making integration easier. Learn how to set up Sparkplug B in Node-RED for efficient data management in industrial environments.
+date: 2024-08-21
+authors: ["sumit-shinde"]
+image: 
```

Comment:
@sumitshinde-84 Please make a tile

Reply:
Done !

Reply:
we need to replace the diagrams !

Reply:
> we need to replace the diagrams !

Are you saying the article wasnt 100% ready @sumitshinde-84?

FWIW, i typically switch a PR to draft if not ready to go.

Reply:
The article is ready; it just needs the diagram updated with the diagrams Yndira will recreate.

Reply:
I created those rough diagrams so that Yndira can understand the type of diagram we need.